translations/messages+intl-icu.fr.xlf

```diff
@@ -434,6 +434,63 @@
                 <source>paginator.next</source>
                 <target>Suivant</target>
             </trans-unit>
+
+            <trans-unit id="action.forgot_password">
+                <source>action.forgot_password</source>
+                <target>Mot de passe oublié</target>
+            </trans-unit>
+            <trans-unit id="reset_password.title">
+                <source>reset_password.title</source>
+                <target>Réinitialiser votre mot de passe</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.check_email.p1">
+                <source>reset_password.check_email.p1</source>
+                <target>Un e-mail contenant un lien sur lequel vous pouvez cliquer pour réinitialiser votre mot de passe a été envoyé. Ce lien expirera dans {token} heure (s).</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.p2">
+                <source>reset_password.check_email.p2</source>
+                <target><![CDATA[Si vous ne recevez pas d'e-mail, veuillez vérifier votre dossier de spam ou <a href="{reset_password_link}">réessayer</a>.]]></target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_exists">
+                <source>reset_password.check_email.help_user_exists</source>
+                <target>Dans une application réelle, l'e-mail ci-dessous serait envoyé et l'utilisateur serait redirigé vers la route app_check_email.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_not_found">
+                <source>reset_password.check_email.help_user_not_found</source>
+                <target>Étant donné que l'adresse e-mail que vous avez saisie n'existe pas, nous affichons ce modèle, au lieu de révéler l'utilisateur n'existe pas.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.title">
+                <source>reset_password.check_email.title</source>
+                <target>Réinitialiser le mot de passe E-mail envoyé</target>
```

Comment:
Email de réinitialisation de mot de passe envoyé

Reply:
I went with "E-mail de réinitialisation de mot de passe envoyé" as I believe that better reflects what is generated by maker.